### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by IATFLG** [Jeff-Zilence/TransGeo2022#20]
The website of the cvusa data set has no effect,  would you be willing to share the data?

**Comment by Jeff-Zilence** [comment]
You can ask the authors of CVUSA (https://mvrl.cse.wustl.edu/datasets/cvusa/) for help. I do not have the right to release other people's datasets, but you can let me know about the issues. You are also free to use the VIGOR dataset (https://github.com/Jeff-Zilence/VIGOR). Btw, the accuracy on CVUSA is almost 100% and it might be better to work on other datasets.

**Comment by IATFLG** [comment]
Thank you for your reply. I have another question. I only have one card. During the training process, there was a memory overflow and the process was killed. My server memory is 16g, not the graphics memory overflow. Is it due to Pytorch code or larger hardware required

**Comment by Jeff-Zilence** [comment]
It should be the "accuracy" function in “train.py” that uses large memory. You can check the function and use a smaller size of queries. Maybe you can fit it in.

**Comment by Jeff-Zilence** [comment]
I will close the issue if there is no further question.

**Comment by IATFLG** [comment]
Thank you for your reply. It has been very helpful to me。The worker thread for data loading can only be set to 0, otherwise it will cause a crash. Is there any solution

**Comment by Jeff-Zilence** [comment]
We use the standard distributed dataloader from Pytorch. You may try using the normal data loader to replace the distributed one.

**Comment by IATFLG** [comment]
When training deep learning network models, the large-scale image datasets used (such as tens of thousands, hundreds of thousands, millions, etc.) contain damaged or damaged images. After a few hours of first iteration, the dataset loading problem suddenly interrupted, resulting in high time costs. How did you solve this problem

**Comment by Jeff-Zilence** [comment]
We did not encounter this issue, but you can write an exception catcher in the data loader.

**Comment by Jeff-Zilence** [comment]
I will close the issue if there is no further question.
